prow/cmd/deck/main.go

```diff
@@ -1488,7 +1696,12 @@ func handleRerun(prowJobClient prowv1.ProwJobInterface, createProwJob bool, cfg
 			}
 			return
 		}
-		newPJ := pjutil.NewProwJob(pj.Spec, pj.ObjectMeta.Labels, pj.ObjectMeta.Annotations)
+		newPJ, err := newRerunProwJob(mode, cfg, pj)
+		if err != nil {
+			http.Error(w, fmt.Sprintf("Could not create new projob: %v", err), http.StatusInternalServerError)
+			l.WithError(err).Errorf("Could not create new projob")
```

Comment:
there is not formatting, so should be `.Error`

Reply:
Fixed!